scripts/generate-alerting-rtk-apis.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * To generate alerting k8s APIs, run:
+ * `npx rtk-query-codegen-openapi ./scripts/generate-alerting-rtk-apis.ts`
+ */
+
+import { ConfigFile } from '@rtk-query/codegen-openapi';
+import { accessSync } from 'fs';
+
+const schemaFile = '../data/alerting/openapi.json';
+
+try {
+  // Check we have the OpenAPI before generating alerting RTK APIs,
+  // as this is currently a manual process
+  accessSync(schemaFile);
+} catch (e) {
+  console.error('\nCould not find OpenAPI definition.\n');
+  console.error(
+    'Please visit /openapi/v3/apis/notifications.alerting.grafana.app/v0alpha1 and save the OpenAPI definition to data/alerting/openapi.json\n'
+  );
+  throw e;
+}
+
+const config: ConfigFile = {
+  schemaFile,
+  apiFile: '',
+  tag: true,
+  outputFiles: {
+    '../public/app/features/alerting/unified/openapi/timeIntervalsApi.gen.ts': {
+      apiFile: '../public/app/features/alerting/unified/api/alertingApi.ts',
+      apiImport: 'alertingApi',
+      filterEndpoints: ['listTimeIntervalForAllNamespaces'],
```

Comment:
Switched to explicitly mentioning the endpoints here, as it
* makes it clearer when a given PR adds a reliance on a new endpoint
* we end up with less code that's potentially unused
* searching through the codebase is more likely to bring up this file, leading to better understanding of how the code is created